apps/tour-of-heroes-ngxs/src/app/createPropertySelector.ts

```diff
@@ -0,0 +1,26 @@
+import { createSelector } from '@ngxs/store';
```

Comment:
This is a utility for creating the basic selectors for your state. We are currently testing with some core users for refinement before adding to the library

Reply:
Ver nice!!!